test/test_bandedlu.jl

```diff
@@ -24,11 +24,7 @@ let
     @test BandedMatrices._promote_to_blas_type(typ, Int16)      == typ
     @test BandedMatrices._promote_to_blas_type(typ, Int8)       == typ
 
-    if VERSION < v"0.5.0-dev+5122"
```

Comment:
I guess technically you could leave this as a version range that only includes a range of -dev and -rc, in case someone needs to bisect on a base Julia bug that involves this package